packages/terra-modal-manager/src/ModalManager.jsx

```diff
@@ -190,20 +192,19 @@ class ModalManager extends React.Component {
 
   render() {
     const { closeModal, size, isFocused, isOpen, isMaximized } = this.props;
-
-    const modalClassNames = classNames([
-      'terra-ModalManager-modal',
-      { [`terra-ModalManager-modal--${size}`]: !(isMaximized || this.forceFullscreenModal) },
+    const isFullscreen = isMaximized || this.forceFullscreenModal;
+    const modalClasses = cx([
+      { [`${styles[size]}`]: !isFullscreen },
     ]);
 
     return (
-      <div className="terra-ModalManager">
+      <div className={styles.container}>
```

Comment:
Because you are already binding the styles above, you could use `className={cx('container')}` here.